packages/flame_tiled/lib/src/tile_transform.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:ui' as ui;
```

Comment:
```suggestion
import 'dart:ui';
```
This shouldn't be clashing with anything else in here?

Reply:
Right. I just blindly copy pasted it 😄